test/servers.yaml

```
@@ -1,28 +1,29 @@
 ---
 - name: sourcegraph-e2e
   box: google/gce
-  machine_type: "custom-16-20480"
+  machine_type: 'custom-16-20480'
   project_id: sourcegraph-ci
   external_ip: false
   use_private_ip: true
   network: default
   username: buildkite
-  ssh_key_path: "~/.ssh/id_rsa"
+  ssh_key_path: '~/.ssh/id_rsa'
   shell_commands:
     - |
-       cd /sourcegraph
-       ./test/e2e/test.sh
+      cd /sourcegraph
+      sleep 3h
+    #    ./test/e2e/test.sh
```

```suggestion
    ./test/e2e/test.sh
```

This is going to be removed, only adding like this for testing.